foxtrot-common/src/main/java/com/flipkart/foxtrot/common/query/FilterOperator.java

```diff
@@ -22,24 +22,24 @@
  */
 public interface FilterOperator {
     //All
-    public static final String equals = "equals";
-    public static final String not_equals = "not_equals";
-    public static final String any = "any";
-    public static final String in = "in";
-    public static final String exists = "exists";
-    public static final String missing = "missing";
+    String equals = "equals";
```

Comment:
Eh?!! Why are these not static final? These are fixed strings.. You're basically screwing the compiler...